desc/integrals/quad_utils.py

```diff
@@ -190,14 +184,67 @@ def leggauss_lob(deg, interior_only=False):
     return x, w
 
 
+def chebgauss_uniform(deg):
+    """Gauss-Chebyshev quadrature with uniformly spaced nodes.
```

Comment:
Not sure I totally understand this. The points are linearly spaced, and the weights are uniform, so isn't this just 1st order reimann sum?

Reply:
yes assuming the riemann sum starts and ends at the correct endpoints, it is the same as gaussian quadrature in a transformed variable y

Reply:
There's a longer comment on the ripple branch, but put another way simply, all quadrature tricks can be stated in the most general sense to make your function as smooth as possible before integrating. This ensures that a sample of the integrand at one point is the best approximation to stuff around it. Hence equivalently you can increase observed smoothness by putting more nodes in non-smooth areas of the function.

Secondary to the smoothness objective is the objective to minimize interpolation errors that result from weaknesses in the choice of basis functions. A Gaussian quadrature in x = 1/pitch sacrifices the first goal, for the secondary goal: minimizing runge. If the quantity to integrate is sufficiently smooth to begin with, then this choice is fine. Neither Gamma_c nor epsilon effective were sufficiently smooth for this. For the former it's easy to see from the expression, for the latter I can only claim that's what i see empirically